automation/unit-tests.sh

```diff
@@ -24,7 +24,7 @@ done
 echo "Processing all templates to find syntax issues"
 for template in $templates; do
   if [[ $template =~ .*saphana.* ]]; then
-    oc process -f "$template" NAME=test TARGET_NODE_NAME=mynode || exit 1
+    oc process -f "$template" NAME=test TARGET_NODE_SELECTOR='{"mynode/lala": "blub"}' SRIOV_NETWORK_NAME1="default/sriov1"  SRIOV_NETWORK_NAME2="default/sriov1"  SRIOV_NETWORK_NAME3="default/sriov1"|| exit 1
```

Comment:
@rnetser I think one caveat of this approach is that when we do not pre-define certain aspects of the template, that people have to provide here a valid json expression.

Another option would be solely allow setting a key, or just a value of a label. I am not sure if the UI can handle well what I am doing here.

Reply:
I think that we should provide the simplest way for users to set the label.
Is it a legitimate requirement to ask to set the label in a specific format, for example:
` node-role.kubernetes.io/sap: "" ` and then only parametrize the suffix of the template key (`node-role.kubernetes.io/${NODE_LABEL}: "" `)?

Reply:
That sounds good. One more thing came to my mind. We will also have to give people the chance to reserve the whole node for hana. That means that people have to taint nodes to avoid that other workloads are going there. I will add a configurable toleration to the template too.

Reply:
Ack